### ISSUE TITLE
Add option to export `endpoints` by default.

### ISSUE THREAD

**Issue by ivenuss** [astahmer/openapi-zod-client#81]
Hi,

I would like to configure the `axiosConfig`, but I'm not able to do that without touching the generated file, which I don't want to. I want to keep it as source of truth to prevent future overrides. 

This the solution that I came up with, but requires `export`ing the `endpoints` variable. Is there any better solution?

```ts
import { Zodios } from '@zodios/core';
import { ZodiosHooks } from '@zodios/react';
import { endpoints } from './generated'; // Not possible

const baseUrl = 'url';

const zodios = new Zodios(baseUrl, endpoints, {
  axiosConfig: {  ...  }
});

export const api = new ZodiosHooks('api', zodios);
```

**Comment by astahmer** [comment]
- you can do so by using a custom template, you can try it [here](https://openapi-zod-client.vercel.app/?activeInputIndex=1)

`-t, --template <path>     Template path for the handlebars template that will be used to generate the output`


- also, irrc `endpoints` are available under the `api` property of every zodios client, which means you can do :
```ts
import { api } from './generated'; // Not possible

const endpoints = api.api; // a bit weird but name clashes here 🤷 
```

![Screenshot 2023-01-30 at 09 51 21](https://user-images.githubusercontent.com/47224540/215430435-d600fe21-cb2b-4ecb-964a-97a29c387b60.png)

- finally, we could just add an option for the 3rd arg in the exported fn `createApiClient`

![Screenshot 2023-01-30 at 09 52 35](https://user-images.githubusercontent.com/47224540/215430718-105e0b51-15c1-4703-838a-be6adaef0f46.png)

**Comment by astahmer** [comment]
3rd arg for `createApiClient` just merged in [1.5.3](https://github.com/astahmer/openapi-zod-client/pull/83)

**Comment by ivenuss** [comment]
Glad to hear that the new `createApiClient` comes with this option. This totally solves my problem.

Thanks!
